powerline/bindings/lemonbar/powerline-lemonbar-multimonitor.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Powerline BAR bindings.')
+    parser.add_argument(
+        '--i3', action='store_true',
+        help='Subscribe for i3 events.'
+    )
+    args = parser.parse_args()
+
+    themes = {
+        'LVDS1': 'default',
+        'HDMI1': 'hdmi',
+        'VGA1': 'hdmi'
+    }
+
+    active_screens = [
+        dict(zip(['name', 'width', 'height', 'x', 'y'], values))
+        for values in re.findall(
+            '^(\w+) connected (\d+)x(\d+)\+(\d+)\+(\d+)',
+            subprocess.check_output('xrandr -q', shell=True).decode(),
+            re.MULTILINE
+        )
+    ]
+
+    powerlines = []
+
+    for screen in active_screens:
+        theme_name = 'default'
+        if 'HDMI1' in screen['name']:
+            theme_name = 'hdmi'
+        elif 'DVI' in screen['name'] or 'VGA' in screen['name']:
+            theme_name = 'dvi'
+
+        line = LemonbarPowerline(theme=theme_name, output=screen['name'])
+        process = subprocess.Popen('bar-aint-recursive -g {}x16+{} -f "Source Code Pro for Powerline-9,Icon-9"'.format(screen['width'], screen['x']), shell=True, stdin=subprocess.PIPE)
```

Comment:
Do not use subprocess.Popen with string argument. Only list arguments are accepted.